refgram/syntax/index.html

```diff
@@ -1659,7 +1659,7 @@ <h4 id="pronominal_classes"><a href="#pronominal_classes">Pronominal classes</a>
 		<div class="vocabularyg">/hoq</div>
```

Comment:
The Deranı for **hóq** should go here:
```suggestion
		<div class="vocabularyg">󱛆󱛊󱛃󱛂</div>
```